components/Workspaces/TrackedRepoWizardStep1.tsx

```diff
@@ -0,0 +1,69 @@
+import { FaSearch } from "react-icons/fa";
+import { BiBarChartAlt2 } from "react-icons/bi";
+import { FaArrowLeft } from "react-icons/fa6";
+import Card from "components/atoms/Card/card";
+import Button from "components/atoms/Button/button";
+
+interface TrackedRepoWizardProps {
+  onAddToTrackingList: () => void;
+  onSearchRepos: () => void;
+  onImportOrg: () => void;
+  onCancel: () => void;
+  trackedReposCount: number;
+}
+
+export const TrackedRepoWizard = ({
+  onAddToTrackingList,
+  onSearchRepos,
+  onImportOrg,
+  onCancel,
+  trackedReposCount,
+}: TrackedRepoWizardProps) => {
+  return (
+    <Card className="p-0 w-max max-w-3xl">
```

Comment:
The text was not passing colour contrast checks as it's below light-slate-12, so I'm currently using the default text colour defined on the body.